src/templates/case-study.scss

```
@@ -0,0 +1,451 @@
+      }
+    }
+  }
+}
+
+.case-page__highlights {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  flex-shrink: 0;
+  border-radius: 16px;
+  box-shadow: 0px 8px 16px 0px rgba(0, 45, 55, 0.12);
+  background: var(--bg-white);
+  padding:40px;
+  
+  &--body {
+    .case-page__column--intro {
+      color: var(--text-primary);
+      font-family: Noto Sans;
+      font-size: 16px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 24px;
+      margin: 0px;
+      margin-bottom: 8px;
+
+      @include m.breakpoint(v.$tablet-sm-exact) {
+        // margin-top: -16px;
+        font-size: 20px;
+        line-height: 32px;
+      }
+
+      @include m.breakpoint(v.$desktop-sm) {
+        // margin-top: -26px;
+      }
+    }
+
+    .case-page__column--title {
+      @include m.font-poppins(v.$fw-bold);
+      @include m.font-scale(x2-large);
+  
+      color: var(--text-service);
+      font-size: 20px;
+      font-style: normal;
+      font-weight: 600;
+      line-height: 32px;
+      margin-top: 0px;
+      margin-bottom: 24px;
+    }
+  
+    > ul {
+      margin-bottom: 40px;
+      padding-left: 16px;
+      gap: 8px;
+      list-style-type: none;
+  
+      li {
+        color: var(--text-primary);
+        position: relative;
+        padding-left: 20px;
+        font-size: 16px;
+        font-style: normal;
+        font-weight: 400;
+        line-height: 24px;
+        margin-bottom: -8px;
+
+        @include m.breakpoint(v.$tablet-sm-exact) {
+          font-size: 20px;
+          line-height: 32px;
+        }
+    
+        &::before {
+          content: '';
+          position: absolute;
+          top: 8px;
+          left: 0;
+          width: 8px;
+          height: 8px;
+          border-radius: 50%;
+          background-color: var(--text-light-grey);
+
+          @include m.breakpoint(v.$tablet-sm-exact) {
+            top: 14px;
+          }
+        }
+      }
+    }
+  
+    > img {
+      margin-bottom: 40px;
+      object-fit: cover;
+    }
+  }
+}
+
+.case-page__benefitsResults {
+  &--body {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+
+    .case-page__column--intro {
+      color: var(--text-primary);
+      font-family: Noto Sans;
+      font-size: 16px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 24px;
+      margin: 0px;
+      margin-bottom: 8px;
+
+      @include m.breakpoint(v.$tablet-sm-exact) {
+        // margin-top: -16px;
+        font-size: 20px;
+        line-height: 32px;
+      }
+
+      @include m.breakpoint(v.$desktop-sm) {
+        // margin-top: -26px;
+      }
+    }
+
+    .case-page__column--title {
+      @include m.font-poppins(v.$fw-bold);
+      @include m.font-scale(x2-large);
+  
+      color: var(--text-service);
+      font-size: 20px;
+      font-style: normal;
+      font-weight: 600;
+      line-height: 32px;
+      margin-top: 0px;
+      margin-bottom: 10px;
```

Please use only mixins for removed styles

```suggestion
      color: var(--text-service);
      margin-top: 0px;
      margin-bottom: 10px;
```

---

Fixed